### ISSUE TITLE
Admin page table link text missing descriptive text for screen readers

### ISSUE THREAD

**Issue by mnigh** [GCTC-NTGC/gc-digital-talent#3701]
Some Admin pages missing descriptive text for screen readers.

### Screenshots
![Screen Shot 2022-08-18 at 10 45 39](https://user-images.githubusercontent.com/3046459/185426498-92d82a1f-6c9b-419c-b9d3-4779cc204b6c.png)
![Screen Shot 2022-08-18 at 10 45 13](https://user-images.githubusercontent.com/3046459/185426500-e5d61d2e-65a8-40d1-bcd1-b9e46c208dfd.png)
![Screen Shot 2022-08-18 at 10 46 03](https://user-images.githubusercontent.com/3046459/185426504-747fc332-4f13-4b7d-bdc6-a504b352c154.png)
![Screen Shot 2022-08-18 at 10 42 33](https://user-images.githubusercontent.com/3046459/185426507-bfd9b598-f897-455c-acfe-5ad43d07f98e.png)
![Screen Shot 2022-08-18 at 10 44 58](https://user-images.githubusercontent.com/3046459/185426508-333cdf1f-f10b-4b85-825a-128f2b2d8aeb.png)
![Screen Shot 2022-08-18 at 10 44 12](https://user-images.githubusercontent.com/3046459/185426509-74a14c85-38b5-4c52-bf51-11d17f668c36.png)

**Comment by petertgiles** [comment]
Don't forget translations

**Comment by petertgiles** [comment]
Even better: "Edit the skill [SKILL NAME]"
